modules/Core/includes/endpoints/CreateReportEndpoint.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @param string $reporter The NamelessMC username of the user who is creating the report
+ * @param string $reported The NamelessMC username of the user who is getting reported
+ * @param string $content The content of the report
+ * 
+ * @return string JSON Array
+ */
+class CreateReportEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'createReport';
+        $this->_module = 'Core';
+        $this->_description = 'Create a report';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Can we remove the type?

Reply:
As per our conversation on Discord, leaving this.